cluster/addons/fluentd-gcp/fluentd-gcp-configmap.yaml

```diff
@@ -108,8 +110,8 @@ data:
       @type detect_exceptions
 
       remove_tag_prefix raw
-      message log
-      stream stream
+      message message
```

Comment:
Seems GitHub is not showing the latest `kubernetes:master` code. `message log` has already been [fixed](https://github.com/kubernetes/kubernetes/blob/master/cluster/addons/fluentd-gcp/fluentd-gcp-configmap.yaml#L111) in https://github.com/kubernetes/kubernetes/commit/8f340c6c6a137b968fd7fc2e730c7dcdf667d624.

We might need to rebase `bmoyles0117:apply-latest-stackdriver-fixes`.